javascript/advanced_examples/gdx_making_a_chart.html

```diff
@@ -42,58 +35,56 @@
 
 	<script>  
  
-	window.chartColors = {
-		// create colors for chart to use
-		green: 'rgb(255, 99, 132)',
-	}; 
-	
-   
-	const selectDeviceBtn = document.querySelector('#select_device');
-    const output = document.querySelector('#output');
-	// create global array for sensor readings to be stored
-    let sensorReadings = [];   
+		window.chartColors = {
```

Comment:
Is there a reason for this to be on the window?

Reply:
I am not sure. I will need to look into desmos to figure it out